### PR TITLE
Kor 76 player spells ui improvement

### PR DESCRIPTION
**Pull request by AlonLivne**

### CODE REVIEW

File: Assets/Scripts/Managers/SpellConfigurationManager.cs [hunk 1]
```diff
@@ -15,6 +15,9 @@ public class SpellConfigurationManager : MonoBehaviour {
     public int MassHealSpellLvl;
     public int SingleHealSpellLvl;
     public int FireBallSpellLvl;
+    public float Spell1CoolDown;
```
Comment (yoavhops) [inline Assets/Scripts/Managers/SpellConfigurationManager.cs]: rename.. fire ball cool down..  or what ever spell it is

File: Assets/Scripts/Spells/PlayerSpells.cs [hunk 2]
```diff
@@ -39,6 +39,10 @@ private void Update()
 
     public virtual void OnUpdate()
     {
+        if (InputManager.GetMouseButtonUp(1))
+        {
+            GameManager.Singleton.ReturnToNormal();
```
Comment (yoavhops) [inline Assets/Scripts/Spells/PlayerSpells.cs]: call whenDestryoed.. and move  GameManager.Singleton.ReturnToNormal(); from Execute
to WhenDestroyed();

File: Assets/Scripts/Managers/SpellConfigurationManager.cs (hunk 1 above)
Comment (AlonLivne) [reply Assets/Scripts/Managers/SpellConfigurationManager.cs]: I did it on purpose as I feel the spells itselves are under development and I don't want the code to depend on spell type

File: Assets/Scripts/Spells/PlayerSpells.cs (hunk 2 above)
Comment (AlonLivne) [reply Assets/Scripts/Spells/PlayerSpells.cs]: Done on Ghost4, but differently

**Pull request merged**